scripts/variantstore/README.md

```diff
@@ -0,0 +1,131 @@
+
+# NOTE: this is a work in progress, please send feedback to dsp-spec-ops@broadinstitute.org
+
+# Before you start
+
+Make sure you have the following ready:
+
+- A collection of HG38 reblocked GVCFs
+- A google project containing
+  - A BigQuery dataset to hold the genomes
+  - A BigQuery dataset for temporary storage (can be same as above, or can be a separate dataset with different permissions or table expiration defaults)
+  - A Google Cloud Storage bucket
+- A cromwell instance running as a user with access to the BigQuery dataset and the google buckets
+- A sample mapping file (see below)
+- Python 3.6+ installed with the bigquery client library installed (`google-cloud-bigquery`)
+
+Note:  Where a parameter is prefixed with `fq` it is a fully qualified name (e.g. `project.dataset` or `project.dataset.table`)
+
+# QuickStart
+
+## Overview
+
+There are 3 primary steps use cases in the Variant Store:
+
+1. Importing Genomes
+2. Training a Filtering Model
+3. Extracting a Cohort with filtering applied
+
+The workflow for each of these is encapsulated in a Cromwell WDL workflow.  These workflows can be found at
+
+ `https://github.com/broadinstitute/gatk/tree/ah_var_store/scripts/variantstore/wdl`
+
+## Importing Genomes
+
+gVCFs from genomes are imported through the `ImportGenomes.wdl` WDL.  An example inputs json for this workflow can be found at `ImportGenomes.example.inputs.json`
+
+Most of the parameters in the WDL are constant, such as the reference genome.  However, several should be changed for each run.  These are:
+
+| Parameter Name| Description |
+| ------------- | ------------- |
+| ImportGenomes.project_id  | The name of the google project containing the BigQuery Dataset  |
+| ImportGenomes.dataset_name  | The name of the BigQuery dataset  |
+| ImportGenomes.sample_map | The GCS Path to the sample map file |
+| ImportGenomes.input_vcfs| An array of the reblocked gVCF files to be loaded |
+| ImportGenomes.output_directory | A GCS path to be used for temporary storage of data while loading |
+
+### Application Notes
+- After a successful import the temporary directory specified by `output_directory` can be deleted.  It is not deleted by default currently by the WDL
+
+
+
+## Training Filtering Model
```

Comment:
we should put the instructions for creating the alt allele table before this

Reply:
+1, I'm stuck on this step right now. Not sure where to pull the alt allele table from. @kcibul @ahaessly

Reply:
@ericsong this is the basic sql to create the alt_allele table, but we haven't written to handle multiple vet tables yet. how many vet tables do you have? let me know if you don't have permissions.
https://console.cloud.google.com/bigquery?sq=365614957255:abe6579d90f4490596733b982c740b19

Reply:
thanks. I only have 1 vet table so that should be fine. I don't think I have access to the query though, nothing showing up when I pulled up that page. @ahaessly